api/src/main/java/org/openmrs/module/pihcore/identifier/mexico/ConfigureMexicoIdGenerators.java

```diff
@@ -38,7 +38,7 @@ private static void createPrimaryIdentifierGenerator(IdentifierSourceService ide
         chiapasPrimaryIdentifierSource.setName("Chiapas Primary Identifier Source");
         chiapasPrimaryIdentifierSource.setDescription("Primary Identifier Generator for Chiapas");
         chiapasPrimaryIdentifierSource.setIdentifierType(MetadataUtils.existing(PatientIdentifierType.class,
-                MexicoPatientIdentifierTypes.CHIAPAS_EMR_ID.uuid()));
+                CesConfigConstants.PATIENTIDENTIFIERTYPE_CHIAPASEMRID_UUID));
```

Comment:
Nit, but I'm wondering if it would make sense to have an underscore in the constants based on both camel case and the level in the yaml file?  (If I'm understanding it, it's only the level in the yaml file now?)

Probably fine how it is, just throwing it out there.

Reply:
Yeah.  We'd have to update the plugin and then update all of the constant references.  If it's important to us, we can do so, but let's punt.

Reply:
Agreed, let's punt!